dcpy/lifecycle/builds/plan.py

```diff
@@ -248,7 +270,47 @@ def _cli_wrapper_plan_recipe(
         False, "--repeat", help="Repeat specific published build"
     ),
 ):
-    plan(recipe_path, version, repeat)
+    plan(recipe_path, version)
+
+
+@app.command("repeat")
+def _cli_wrapper_repeat_recipe(
+    product: str = typer.Argument(help="Name of the product to build"),
+    product_type: str = typer.Option(
+        None, "--product-type", "-t", help="Product/build type ('draft' or 'publish')"
+    ),
```

Comment:
(just a thought, not worth holding this up for)

this seems more like `version_type` since it's used to pick a type of `ProductKey`

Reply:
Yeah this was a tough one to name. Literally, more "product instance type" but that doesn't roll off the tongue or keyboard